params/config_arbitrum.go

```diff
@@ -38,6 +39,28 @@ func (c *ChainConfig) DebugMode() bool {
 	return c.ArbitrumChainParams.AllowDebugPrecompiles
 }
 
+func (c *ChainConfig) checkArbitrumCompatible(newcfg *ChainConfig, head *big.Int) *ConfigCompatError {
+	boolToBig := func(b bool) *big.Int {
+		if b {
+			return common.Big1
+		}
+		return common.Big0
+	}
+
+	if c.IsArbitrum() != newcfg.IsArbitrum() {
+		return newCompatError("isArbitrum", boolToBig(c.IsArbitrum()), boolToBig(newcfg.IsArbitrum()))
```

Comment:
I believe both arguments in the `newCompatError` should be `common.Big0`, as if a chain is Arbitrum is determined at the genesis. The arguments to `newCompatError` do not need to necessarily differ -- see the chain ID check for instance.